mdoc/Mono.Documentation/Updater/DocUtils.cs

```diff
@@ -200,6 +200,13 @@ public static bool IsDelegate (TypeDefinition type)
                     baseRef.FullName == "System.MulticastDelegate";
         }
 
+        public static bool NeedOverwrite(XmlElement element)
+        {
+            return element != null &&
+                   (!element.HasAttribute("overwrite") ||
+                    element.Attributes["overwrite"].Value != "false");

```

Comment:
Please change this to use a case insensitive check, that way, in case a user ever uses `"False"`, it will still work